File: dener8/programmers_짝지어제거하기.py
def solution(s):
    answer = 1
    stack = []
    for alph in s:
        stack.append(alph)
        if len(stack) >= 2 and stack[-1] == stack[-2]:
            stack.pop()
            stack.pop()
    if stack:
        answer = 0

    return answer

File: myunghyunnero/programmers_짝지어제거하기.cpp
#include <iostream>
#include<string>
#include <stack>
using namespace std;
stack<char> stk;
int solution(string s)
{
    int answer = -1;
    int cnt=0;
    for(char c:s){
        if(stk.empty()){
            stk.push(c);
        }else{
            char top = stk.top();
            if(top==c){
                stk.pop();
            }else{
                stk.push(c);
            }
        }
    }
    if(stk.empty()) return 1;
    else return 0;
}